narayana/ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/performance/Performance1.java

```diff
@@ -75,27 +66,37 @@
     }
 
     @Benchmark
-    public boolean onePhase(BenchmarkState benchmarkState) {
-        AtomicAction A = new AtomicAction();
+    public boolean onePhaseTest(BenchmarkState benchmarkState, Blackhole bh) {
+    	bh.consume(onePhase(benchmarkState));
 
+        return true;
+    }
+
+    private boolean onePhase(BenchmarkState benchmarkState) {
+    	AtomicAction A = new AtomicAction();
+        
         A.begin();
         A.add(benchmarkState.record1);
         A.commit();
+		return true;
+	}
```

Comment:
Just a remark but I enable checkstyle in my IDE which shows up quite a few violations. I raised [an issue](https://issues.redhat.com/browse/JBTM-3258) to include it in the performance repo.

Until we have that the policy to be followed when changing a file is to follow the existing style used in the file you are modifying. In particular the files affected by this PR use spaces but I notice that you have introduced tab characters in some places. Please could you replace the tabs you introduced with space characters. This comment also applies to some of the other files that you changed.

Reply:
So this checkstyle issue, will I have to fix as part of https://issues.redhat.com/browse/JBTM-3258 ?

Reply:
Just the tabs